### PR TITLE
feat(logbook-link): Update for Logbook

### LINKED ISSUE #2391: [Web] - Update for Logbook
Tasks:
- `...` entries
- Avatar entry

### PR DESCRIPTION
**Pull request by tx0c**
Update for Logbook resolves #2391

### CODE REVIEW

File: .env.dev [hunk 1]
```diff
@@ -22,4 +22,5 @@ NEXT_PUBLIC_STRIPE_PUBLIC_KEY=pk_test_51GrOGRCE0HD6LY9Uo7rK3pSmiIG4KTjWO1rOrJavF
 NEXT_PUBLIC_BUILD_TYPE=dynamic
 NEXT_PUBLIC_PROGRAMMABLE_SEARCH_ENGINE_ID=004538121411474993797:xkl3sdy-9su
 NEXT_PUBLIC_TRAVELOGGERS_URL=https://nft-develop.matters.news
+NEXT_PUBLIC_LOGBOOKS_URL=https://logbooks-dev.eth.link
```
Comment (robertu7) [inline .env.dev:25]: LOGBOOK or LOGBOOKS?

Comment (tx0c) [reply .env.dev:25]: because of `NEXT_PUBLIC_TRAVELOGGERS_URL` ?

File: src/components/UserProfile/LogbookDialog/index.tsx [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  children,
+}) => {
+  const { lang } = useContext(LanguageContext)
+  const { show, openDialog, closeDialog } = useDialogSwitch(false)
+
+  // const address = user.info.cryptoWallet?.address
+  const logbook1Url = `${process.env.NEXT_PUBLIC_TRAVELOGGERS_URL}${
+    lang === 'en' ? '/' : '/zh/'
+  }owner/${address}`
+  const logbook2Url = `${process.env.NEXT_PUBLIC_LOGBOOKS_URL}/owner/${address}`
```
Comment (tx0c) [inline src/components/UserProfile/LogbookDialog/index.tsx:31]: am not seeing a full url scheme yet, of the logbooks site url path design,

File: src/components/UserProfile/LogbookDialog/index.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+          </Form.List>
+          <style jsx>{`
+            p {
+              margin-bottom: var(--spacing-loose);
+              text-align: center;
+              color: var(--color-grey-darker);
+            }
+            :global(.group + .group) {
+              margin-top: var(--spacing-loose);
+            }
```
Comment (tx0c) [inline src/components/UserProfile/LogbookDialog/index.tsx:99]: not sure if @robertu7 has better idea to show the space between?

Comment (robertu7) [reply src/components/UserProfile/LogbookDialog/index.tsx:99]: `<Spacer />` ?

File: src/components/UserProfile/LogbookDialog/index.tsx [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+              zh_hant="My Logbook"
+              zh_hans="My Logbook"
+            />
+          }
+          closeDialog={closeDialog}
+          closeTextId="close"
+        />
+        <Dialog.Content spacing={['xloose', 'base']}>
+          <p>
+            Logbook 2.0 has just launched. If you are owner of Traveloggers,
```
Comment (robertu7) [inline src/components/UserProfile/LogbookDialog/index.tsx:50]: translations?

File: src/components/UserProfile/LogbookDialog/index.tsx (hunk 2 above)
Comment (robertu7) [reply src/components/UserProfile/LogbookDialog/index.tsx:31]: `/bookcase?address=0x...`

File: src/components/UserProfile/LogbookDialog/index.tsx [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+  const { show, openDialog, closeDialog } = useDialogSwitch(false)
+
+  // const address = user.info.cryptoWallet?.address
+  const logbook1Url = `${process.env.NEXT_PUBLIC_TRAVELOGGERS_URL}${
+    lang === 'en' ? '/' : '/zh/'
+  }owner/${address}`
+  const logbook2Url = `${process.env.NEXT_PUBLIC_LOGBOOKS_URL}/bookcase/?address=${address}`
+  const [
+    hasClaimedLogbook2,
+    // setClaimed
```
Comment (tx0c) [inline src/components/UserProfile/LogbookDialog/index.tsx:38]: the `setClaimed` is not implemented yet, may add this logic later;

this is no harm for users if already claimed

**Pull request merged**